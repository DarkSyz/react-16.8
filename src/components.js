// this is HEADER file for __components__

export { default as SimpleComponent }
    from './__components__/SimpleComponent';
export { default as CheckBoxWithLabel }
    from './__components__/CheckBox';    
export { Counter }
    from './__components__/CounterWithGlobal';        