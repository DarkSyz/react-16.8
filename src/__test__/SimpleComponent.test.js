// Snapshot test

import React from 'react';
import { render, cleanup } from 'react-testing-library';
import { SimpleComponent } from '../components';

// automatically unmount and cleanup DOM after the test is finished
afterEach(cleanup);

test(`<username/> should render Ben`, ()=>{
    const { container } = render(
        <SimpleComponent username='Ben'/>
    );
    expect(container).toMatchSnapshot();
});
