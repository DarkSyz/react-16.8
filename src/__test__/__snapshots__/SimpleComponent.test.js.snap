// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<username/> should render Ben 1`] = `
<div>
  <h1>
    Hello
  </h1>
  <h1>
    Ben
  </h1>
</div>
`;
